pkg/work/spoke/controllers/finalizercontroller/add_finalizer_controller.go

```diff
@@ -40,12 +42,14 @@ func NewAddFinalizerController(
 
 	return factory.New().
 		WithInformersQueueKeysFunc(queue.QueueKeyByMetaName, manifestWorkInformer.Informer()).
-		WithSync(controller.sync).ToController("ManifestWorkAddFinalizerController", recorder)
+		WithSync(controller.sync).ToController(manifestWorkAddFinalizerController, recorder)
 }
 
 func (m *AddFinalizerController) sync(ctx context.Context, controllerContext factory.SyncContext) error {
 	manifestWorkName := controllerContext.QueueKey()
-	klog.V(5).Infof("Reconciling ManifestWork %q", manifestWorkName)
+	logger := klog.FromContext(ctx).WithName(manifestWorkAddFinalizerController).
```

Comment:
do we have a chance to add the container name to the log context?

Reply:
we might be able to get podName from the env var.